### ISSUE TITLE
Transactions not updated, failed to fetch internal transactions 

### ISSUE THREAD

**Issue by csfl** [poanetwork/blockscout#1231]
*Describe your issue here.*

### Environment

* Elixir & Erlang/OTP versions (`elixir -version`): 1.7
* Operating System: Ubuntu 18

### Steps to reproduce

Testing it with quorum private chain, connected to geth.

### Expected behaviour

Transactions updated in realtime

### Actual behaviour

Transactions are shown in the transaction tab, however, the counter of it and the counter in each block are always 0.

application=indexer [error] failed to fetch internal transactions for 7 transactions: [%{code: -32000, data: %{block_number: 48307, transaction_hash: "0x61a6fd06d89f68eb68488074cf29675b7cbb7d24f702610835c2ae79e58888fb", transaction_index: 0}, message: "(anonymous): Line 7:9 Unexpected token ( (and 160 more errors)"}, %{code: -32000, data: %{block_number: 48308, transaction_hash: "0xbc53c0e5d6c6d99f62a0da7f0d2a8c9ae0f4ef9726cc4cc4114bb419952ae194", transaction_index: 0}, message: "(anonymous): Line 7:9 Unexpected token ( (and 160 more errors)"}, %{code: -32000, data: %{block_number: 48311, transaction_hash: "0x01cc5b8d05383848e342b8ae4b9bd4601752ed31d460c9e793b13a18dcbb6a3c", transaction_index: 4}, message: "(anonymous): Line 7:9 Unexpected token ( (and 160 more errors)"}, %{code: -32000, data: %{block_number: 48311, transaction_hash: "0x06d95dd575f161fc1c84e6c3f6019dc5f98e677dc97aa63ffd1d46c28e5b9a58", transaction_index: 0}, message: "(anonymous): Line 7:9 Unexpected token ( (and 160 more errors)"}, %{code: -32000, data: %{block_number: 48311, transaction_hash: "0x3ff12e0fa2c2f5f01007b4aa017a813172cff624aa70b7211e9cc1e513f16f39", transaction_index: 1}, message: "(anonymous): Line 7:9 Unexpected token ( (and 160 more errors)"}, %{code: -32000, data: %{block_number: 48311, transaction_hash: "0x5bc8f3f4c78ef49d1dd138f439ae1fe76c527820c63081f1d6b3d6e27638c0b1", transaction_index: 2}, message: "(anonymous): Line 7:9 Unexpected token ( (and 160 more errors)"}, %{code: -32000, data: %{block_number: 48311, transaction_hash: "0xa0b8c1d2b594b2af44d18cfcd1cc7d9eef93d159ceb7a8ae5b05622592c9e6ec", transaction_index: 3}, message: "(anonymous): Line 7:9 Unexpected token ( (and 160 more errors)"}]

**Comment by acravenho** [comment]
Hi @csfl,

We have not tested against Quorum yet so there are probably areas of the indexer and import functions that need to be created in order to have full Quorum support. I'll leave this issue open until we can provide full support for Quorum.

**Comment by jaredswarts55** [comment]
This is really old here, but we are running quorum and I am curious if you found work arounds @csfl?

**Comment by jaredswarts55** [comment]
Ignore my previous comment, it looks like our private quorum was having some issues. I am not sure if I will see this error when we stabilize that environment.

**Comment by huayun321** [comment]
> Ignore my previous comment, it looks like our private quorum was having some issues. I am not sure if I will see this error when we stabilize that environment.

have you found how to fix this?
i have the same problem with quorum and blockscout  
code: -32000

**Comment by ellamonroe6047** [comment]
Yes, there’s some kind of givеаway going on. I got some Ethereum there yesterday for different wallets. It’s unlikely they’ll take it back from me, but it’s a pity I didn’t have more wallets and couldn’t get more. But maybe it’s for the best.

<img width="369" alt="Image" src="https://github.com/user-attachments/assets/69225a0e-1078-49c9-b71d-065dce6fe235" />

<img width="373" alt="Image" src="https://github.com/user-attachments/assets/836ca698-1fdb-4573-8e8c-3edaeeb1bdd5" />
